### DISCUSSION TITLE
Why is the Docker container launching a SFTP and SSH Avahi services ?

### DISCUSSION THREAD

**Discussion by jpbaril** [mikebrady/shairport-sync#1921] [Q&A] [answered]
```
Starting shairport-sync
Service "blackbox-ng" (/etc/avahi/services/ssh.service) successfully established.
Service "blackbox-ng" (/etc/avahi/services/sftp-ssh.service) successfully established.
```
On Ubuntu 24.10 (Gnome DE):
![Capture d’écran du 2024-10-12 21-19-57](https://github.com/user-attachments/assets/b660b4af-ab2a-4acc-89e8-cf476accf1ff)
It's Gnome's "Network" view.

I would rather not have it shown there especially that it's does not seem useful and does not work anyway.

**Comment by mikebrady** [comment, marked as answer]
Good catch! Actually this should be an Issue rather than a Q&A. Anyway, I think it is fixed in the `development` branch and should be in the `development`-tagged Docker image.

The cause was that the default Avahi installation includes service definition files for SFTP and SSH. The fix was to remove them.
